platform/windows/display_server_windows.cpp

```diff
@@ -2525,7 +2525,7 @@ Error DisplayServerWindows::dialog_show(String p_title, String p_description, Ve
 
 	Char16String title = p_title.utf16();
 	Char16String message = p_description.utf16();
-	List<Char16String> buttons;
+	Vector<Char16String> buttons;
```

Comment:
Does this need to be `Vector` rather than `LocalVector`?

Reply:
Will look